charts/graphscope/values.yaml

```diff
@@ -70,6 +70,9 @@ engines:
     enabled: False
     image:
       name: dataset
+#    proxy:
```

Comment:
Could you add a comment here for what is `proxy` for, and what's the target scenario, it makes people or developers like me could understand this more easily